### DISCUSSION TITLE
jp2.zip not rendered correctly

### DISCUSSION THREAD

**Discussion by altongusta** [sumatrapdfreader/sumatrapdf#3354] [General]
Zipped jp2 files not rendered correctly. A single non-zipped jp2 file rendered ok. This happens with version 3.4.6 x64. Attached a comparison of 2 renderings and a test file. Image is from [archive.org](https://archive.org/download/northdevoncoast00harp/northdevoncoast00harp_jp2.zip).

carried to issues.

**Comment by kjk** [comment]
Please open an issue for bugs

**Reply by altongusta** [reply]
ok
